app/services/project_data_service.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+
+class ProjectDataService
+
+  def initialize(project, access_token)
+    @project = project
+    @access_token = access_token
+    @client = Octokit::Client.new(access_token: access_token)
+    @circle_config = circle_config
+  end
+
+  def github_org
+    Organization.find(@project.organization_id).name
+  end
+
+  def project_name
+    url = git_remote.split("/").last
+    url.gsub(".git", "")
+  end
```

Comment:
Ditto about this... the line linked above is performing this same operation and they can probably be combined.